### ISSUE TITLE
Restructure repository

### ISSUE THREAD

**Issue by ol-roboro** [openela/openela-documentation#6]
@wsoyinka mentioned a potential restructure of the repositories.

The following items were proposed:

1. Potentially moving content into branches:
- main: containing legal and README and CONTRIBUTING etc
- openela8: for EL8 documentation
- openela9: for EL9 documentation

2. Structuring topics in the branches according to topic areas/groupings. Currently the folders for each part in the ToC might look as follows:

osmanage (or probably better as core)
userauth
software-management
backup
accessibility
cockpit
fsadmin
shareadmin
nbde
firewall
certmanage
openssh
selinux
network
balancing
vpn

This proposal is fully open to discussion and we may prefer to break up folders even further (e.g. systemd might move into its own folder instead of being a component inside osmanage/core.

**Comment by dirkmueller** [comment]
@taroth21 any opinion?

**Comment by dirkmueller** [comment]
we should have LICENSE at least on all the branches, probably also README.md or CONTRIBUTING.md

**Comment by taroth21** [comment]
> @taroth21 any opinion?

I'm fine branches as suggested above

**Comment by ol-roboro** [comment]
I have created the two branches and flattened structure a bit to start to facilitate breaking out topics into specific directories dedicated to topic areas. I have also opened a pull request on an update to main to update the README and CONTRIBUTING and to remove the two doc directories.
I have not started any work on breaking topics into the directories proposed above, as I think Wale was going to look at this.
